tests/foreman/ui/test_remoteexecution.py

```diff
@@ -0,0 +1,238 @@
+"""Test class for Remote Execution Management UI"""
+from robottelo.decorators import run_only_on, stubbed
+from robottelo.test import UITestCase
+
+
+class TestRemoteExecution(UITestCase):
+    """Test class for remote execution feature"""
+
+    @stubbed()
+    @run_only_on('sat')
+    def test_re_create_job_template_1(self):
```

Comment:
Also tests need to have `positive`, `negative` text added in the test case names whenever possible.